primitives/runtime/src/chain.rs

```diff
@@ -319,6 +319,45 @@ macro_rules! decl_bridge_finality_runtime_apis {
 	};
 }
 
+/// Convenience macro that declares bridge GRANDPA finality runtime apis and related constants
+/// for a chain.
+/// This includes:
+/// - chain-specific bridge runtime APIs:
+///     - `<ThisChain>GrandpaFinalityApi`
+/// - constants that are stringified names of runtime API methods:
+///     - `<THIS_CHAIN>_GRANDPA_JUSTIFICATIONS_METHOD`
+/// The name of the chain has to be specified in snake case (e.g. `rialto_parachain`).
+#[macro_export]
```

Comment:
Maybe we could change the existing `decl_bridge_finality_runtime_apis` macro to generate this additional method in the same trait. E.g. for parachains the "call" will be the `decl_bridge_finality_runtime_apis!(bridge_hub_wococo)`, but for standalone/relay chains it'll be something like `decl_bridge_finality_runtime_apis!(kusama, grandpa)`? But feel free to leave as is

Reply:
I added a new API for it for 2 reasons:
1. I don't know if it make sense for parachains to implement it. 
2. There will be a separate implementation (so probably a separate API) for BEEFY

Personally I would keep it as it is for the moment. We can change it in the future if needed.

Reply:
> 1. I don't know if it make sense for parachains to implement it.

That's exactly why I have suggested to add this 2nd arg to existing macro :) So it'll generate trait with single (old) method for parachains and two methods (old and new) for standalone/relay chains :) But ok - let's leave as is - it's fine too

Reply:
Oh, got it. I misread it the first time. Makes sense. Just wondering if it wouldn't be confusing to have the same API with different methods depending on the context. I'm not sure how it would be best. I would still go with a separate API for the beginning. But will keep this in mind and maybe we'll get a better picture as the implementation progresses.

Reply:
Actually looks good. Done.